update-config-files.php

```diff
@@ -0,0 +1,363 @@
+          'emailPrefix' => array('varName' => 'emailPrefix', 'varValue' => '', 'varComment' => 'Email prefix', 'varType' => ''),
+          'emailCC' => array('varName' => 'emailCC', 'varValue' => '', 'varComment' => 'CCed for every email', 'varType' => ''),
+          'emailBCC' => array('varName' => 'emailBCC', 'varValue' => '', 'varComment' => 'BCCed for every email', 'varType' => ''),
+          'phonedbHost' => array('varName' => 'phonedbHost', 'varValue' => '', 'varComment' => '', 'varType' => ''),
+          'phonedbName' => array('varName' => 'phonedbName', 'varValue' => '', 'varComment' => '', 'varType' => ''),
+          'phonedbUser' => array('varName' => 'phonedbUser', 'varValue' => '', 'varComment' => '', 'varType' => ''),
+          'phonedbPass' => array('varName' => 'phonedbPass', 'varValue' => '', 'varComment' => '', 'varType' => ''),
+          'myNewVar' => array('varName' => 'myNewVar', 'varValue' => "'myNewValue'", 'varComment' => 'my new comment', 'varType' => 'static'),
+        );
+        $portalConfigs = rGlob('./db?*config.php');
+        foreach ($portalConfigs as $config)
+        {
+          //make a backup first
+          $date = new Datetime();
+          $timestamp = $date->format('U');
+          $backupConfig = dirname($config) . '/' . basename($config) . '.' . $timestamp . '.php';
+          copy($config, $backupConfig);
+
+          $file = file_get_contents($config);
+          $varArray = parseStr($file);
+          $vars = array_intersect_key(array_merge_recursive($portalConfig, $varArray), $portalConfig);
+          $savedConfig = savePortal($config, $vars);
+          //print_r($savedConfig);
+        }
+    }
+
+    /**
+     * [writeNexusConfig writes/backup all Nexus configs in a root dir]
+     * To add a variable to the config files add it to the array or to remove a variable remove it from the array
+     * Note: if no value is supplied for a variable it will use the one in config file if exists
+     * @return [string] [config content]
+     */
+    function writeNexusConfig()
+    {
+        $nexusConfig = array(
+          'title' => array('varName' => 'title', 'varValue' => '', 'varComment' => '', 'varType' => ''),
+          'city' => array('varName' => 'city', 'varValue' => '', 'varComment' => ''),
+          'adminLogonName' => array('varName' => 'adminLogonName', 'varValue' => '', 'varComment' => 'Administrator\'s logon name', 'varType' => ''),
+          'adPath' => array('varName' => 'adPath', 'varValue' => '', 'varComment' => 'Active directory paths', 'varType' => ''),
+          'uploadDir' => array('varName' => 'uploadDir', 'varValue' => '', 'varComment' => 'Directory for user uploads. using backslashes (/), with trailing slash', 'varType' => ''),
+          'ERM_Sites' => array('varName' => 'ERM_Sites', 'varValue' => "array('')", 'varComment' => 'URL to ERM sites with trailing slash', 'varType' => ''),
```

Comment:
`varValue` should be defaulted to empty string, so that it doesn't set `ERM_Sites` to `array('')`